asciidoc/getting_started/next/init.adoc

```diff
@@ -0,0 +1,65 @@
+== @ensono-stacks/next:init
+
+Adds custom config and developer tools to an existing next application
+
+The next init generator will add a custom ESlint config to an existing NextJs application, install `eslint-plugin-testing-library` to the project. as well as update project.json with a custom test config to allow coverage collection from https://jestjs.io/[jest].
+
+=== Prerequisites
+
+An existing https://nextjs.org/[Next] application
+
+=== Usage
+
+[source, bash]
+nx g @ensono-stacks/next:init
+
+=== Command line arguments
+
+The following command line arguments are available:
+
+[cols="1,1,1,1,1"]
+|===
+|Option |Description | Type | Accepted Values|Default
+
+|--project
+|Name of the existing next application
+|string
+|nameOfApplication
+|N/A
+|===
+
+=== Generator Output
+
+The following files will be updated.
+
+[source, text]
+----
+UPDATE apps/baseline-next-app/project.json #Updated with custom test config to allow for coverage collection
+UPDATE apps/baseline-next-app/.eslintrc.json #Ehanced with additional linting rules
+UPDATE apps/baseline-next-app/tsconfig.json #Minor enhancements
+UPDATE apps/baseline-next-app/tsconfig.spec.json #Updates for monorepo structure
+UPDATE apps/baseline-next-app/specs/index.spec.tsx #Formatting changes
+----
+
+The generator will also add https://github.com/dequelabs/axe-core-npm/blob/develop/packages/react/README.md[react-axe] (version 4.7.3) into the app via the following:
+
+....app.tsx
```

Comment:
could change this to just .app.tsx so it acts as the header for the codeblock. Currently seems a bit out of place when viewed in the browser.